gridstatus/caiso.py

```diff
@@ -1193,30 +1193,54 @@ def _process_tie_flows_data(self, df):
             ],
         ).rename(columns={"MARKET_TYPE": "MARKET", "VALUE": "MW"})
 
-        # Create an identifier column using DIRECTION, FROM_BAA, TO_BAA
-        df["DIRECTION_FROM_BAA_TO_BAA"] = df[["DIRECTION", "FROM_BAA", "TO_BAA"]].apply(
-            lambda x: "_".join(x.astype(str)),
-            axis=1,
+        # Multiply imports by -1 to match convention of imports being negative
+        df["MW"] = np.where(df["DIRECTION"] == "I", df["MW"] * -1, df["MW"])
+
+        # Sum MW by Interval Start, TIE_NAME, FROM_BAA, TO_BAA so we can remove
+        # the direction column
+        df = (
+            df.groupby(
+                [
+                    "Interval Start",
+                    "Interval End",
+                    "TIE_NAME",
+                    "FROM_BAA",
+                    "TO_BAA",
+                    "MARKET",
+                    "BAA_GRP_ID",
+                ],
+            )["MW"]
+            .sum()
+            .reset_index()
+        )
+
+        df.columns = df.columns.map(
+            lambda x: x.title()
+            .replace("_", " ")
+            .replace("Baa", "BAA")
+            .replace("Mw", "MW")
+            .replace("Id", "ID"),
         )
 
+        df["Interface ID"] = df["Tie Name"] + "_" + df["From BAA"] + "_" + df["To BAA"]
```

Comment:
what do you think about using double underscore to separate each part? some of the values have underscores in them, so I think this will help the readability of these ids.

Reply:
That's a good idea though I'm not sure if double underscores or another separate, like a hyphen, would be better.

Reply:
im good with hyphen!